### PR TITLE
[MIG] account_financial_risk: Migration to 14.0

### PR DESCRIPTION
**Pull request by psugne**
@Tecnativa

### CODE REVIEW

**Comment by tonygalmiche** [comment]
in the README file, there is this : 
image

But with runbot i don't have the "Include Sale Orders" line
image

There is an explanation?

**Comment by pedrobaeza** [comment]
Such thing is added by the sale_financial_risk module.

**Review by tonygalmiche — APPROVED** [review]
LGTM : Functional review only with runbot

**Comment by psugne** [comment]
@tonygalmiche can it be merged now?

**Comment by pedrobaeza** [comment]
@psugne 2 reviews are needed at least for a PR to be merged according contributing guidelines and one from a PSC:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#721please-respect-a-few-basic-rules

**Comment by pedrobaeza** [comment]
I'm seeing anyway that you haven't done the procedure depicted in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0 properly, as you have to pass pre-commit just after the commit history preservation. You should squash as well all your migration commits into one. Please do it.

File: account_financial_risk/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_partner_risk_exceeded_wiz_user,Partner Risk Exceeded Wizard (Internal user),model_partner_risk_exceeded_wiz,base.group_user,1,0,0,0
```
Comment (alan196) [inline account_financial_risk/security/ir.model.access.csv]:
```suggestion
access_partner_risk_exceeded_wiz_user,Partner Risk Exceeded Wizard (Internal user),model_partner_risk_exceeded_wiz,base.group_user,1,1,1,1
```

You need more permission to the wizard, this must be fixed in order to raise the wizard

**Review by alan196 — CHANGES_REQUESTED** [review]
Tested functionally and only one change requested

**Review by carlosdauden — APPROVED** [review]
Attend @alan196 request changes

**Comment by pedrobaeza** [comment]
Can you please squash migration commits?

File: account_financial_risk/security/ir.model.access.csv (hunk 1 above)
Comment (psugne) [reply account_financial_risk/security/ir.model.access.csv]: > Attend @alan196 request changes



> Attend @alan196 request changes

Sure! Changes are done.

**Review by alan196 — CHANGES_REQUESTED** [review]
C̶o̶u̶l̶d̶ ̶y̶o̶u̶ ̶p̶l̶e̶a̶s̶e̶ ̶c̶o̶n̶s̶i̶d̶e̶r̶ ̶t̶h̶i̶s̶ ̶P̶R̶ ̶h̶t̶t̶p̶s̶:̶/̶/̶g̶i̶t̶h̶u̶b̶.̶c̶o̶m̶/̶O̶C̶A̶/̶c̶r̶e̶d̶i̶t̶-̶c̶o̶n̶t̶r̶o̶l̶/̶p̶u̶l̶l̶/̶1̶1̶1̶ ̶i̶t̶ ̶j̶u̶s̶t̶ ̶g̶e̶t̶ ̶m̶e̶r̶g̶e̶d̶ ̶y̶e̶s̶t̶e̶r̶d̶a̶y̶ ̶a̶n̶d̶ ̶i̶t̶ ̶a̶f̶f̶e̶c̶t̶s̶ ̶a̶l̶s̶o̶ ̶V̶1̶4̶

**Comment by pedrobaeza** [comment]
But that PR is for another module, isn't it?

**Review by alan196 — APPROVED** [review]
Sorry about that! I get confused 👍

**Pull request merged**